src/main/java/com/fauna/client/RequestBuilder.java

```diff
@@ -51,7 +51,7 @@ static class Headers {
         static final String FORMAT = "X-Format";
     }
 
-    public RequestBuilder(URI uri, String token) {
+    public RequestBuilder(URI uri, String token, int max_contention_retries) {
```

Comment:
A bit of snake case sneaking in...

```suggestion
    public RequestBuilder(URI uri, String token, int maxContentionRetries) {
```